### PR TITLE
Docstring improvement for nx_pylab.py

### PR DESCRIPTION
**Pull request by cnfionawu**
Added docstring for font_color and edgecolors.

### CODE REVIEW

File: networkx/drawing/nx_pylab.py [hunk 1]
```diff
@@ -373,7 +374,10 @@ def draw_networkx_nodes(
         Line width of symbol border
 
     edgecolors : [None | scalar | sequence] (default = node_color)
-        Colors of node borders
+        Colors of node borders. Can be a single color or a sequence of colors with the
+        same length as nodelist. Color can be string or rgb (or rgba) tuple of floats
+        from 0-1. If numeric values are specified they will be mapped to colors
+        using the cmap and vmin,vmax parameters. See matplotlib.scatter for more details.
```
Comment (MridulS) [inline networkx/drawing/nx_pylab.py:380]: We can use intersphinx, to make this even better (I think the following should work, but need to test)
```suggestion
        using the cmap and vmin,vmax parameters. See `~matplotlib.pyplot.scatter` for more details.
```

**Review by dschult — APPROVED** [review]
:)

**Comment by MridulS** [comment]
I think it's fine to merge this one, the CI fail is due to sphinx version mismatch with the main branch.

**Review by rossbar — APPROVED** [review]
> I think it's fine to merge this one, the CI fail is due to sphinx version mismatch with the main branch.

Agreed - and since this originates from the `main` branch on the fork there's no way to push an update to be sure. I tested locally after rebasing on main + with the latest dev env and everything came out okay (no build errors + proposed links were working).

I'll go ahead and merge and we can fix up anything that snuck through in a followup. Thanks @cnfionawu & @MridulS !

**Pull request merged**